src/app/[locale]/(auth)/layout.tsx

```diff
@@ -1,8 +1,21 @@
+import arcjet, { detectBot, request } from '@/libs/Arcjet';
 import { routing } from '@/libs/i18nNavigation';
 import { enUS, frFR } from '@clerk/localizations';
 import { ClerkProvider } from '@clerk/nextjs';
 import { setRequestLocale } from 'next-intl/server';
 
+const aj = arcjet.withRule(
```

Comment:
Is it possible to combine with the one at `src/app/[locale]/(marketing)/layout.tsx`?

And move to the root layout: https://github.com/ixartz/Next-js-Boilerplate/blob/main/src/app/%5Blocale%5D/layout.tsx

So, we keep only one and everything is inside the root layout.

Most pages under (auth) required authentication, so CATEGORY:SEARCH_ENGINE still won't be allowed

Reply:
I did it that way because often you want more restrictive bot detection on the authenticated routes vs your website, but I've made the change. The Arcjet call is now in the root layout. It can easily be moved by the user later.